doc/kafkamanagment/kafka-acl.md

```diff
@@ -28,6 +27,11 @@ To create a Kafka ACL resource you need to prepare the yaml manifest. Here is an
 apiVersion: kafkamanagement.instaclustr.com/v1beta1
 kind: KafkaACL
 metadata:
+  labels:
+    app.kubernetes.io/name: kafkaacl
+    app.kubernetes.io/instance: kafkaacl-sample
+    app.kubernetes.io/part-of: operator
+    app.kubernetes.io/created-by: operator
```

Comment:
I think there is no need for labels in examples

Reply:
I thought that it would be nice to show similar example as in samples files